x-pack/legacy/plugins/siem/public/containers/timeline/index.tsx

```diff
@@ -168,7 +182,15 @@ const makeMapStateToProps = () => {
   return mapStateToProps;
 };
 
-const connector = connect(makeMapStateToProps);
+const mapDispatchToProps = (dispatch: Dispatch) => ({
+  clearSelected: ({ id }: { id: string }) => dispatch(timelineActions.clearSelected({ id })),
+  clearEventsLoading: ({ id }: { id: string }) =>
```

Comment:
I think that I will just add one function like that so you can have one props instead of two and then you can also forget about defining clearSignalsState on line [114](https://github.com/elastic/kibana/pull/56825/files#diff-c1edaa4a709c8299d6dbe9539a5c9d03R114) and just use it from the props.
```
clearSignalState: ({ id} : {id: string }) => {
  if (id === SIGNALS_PAGE_TIMELINE_ID) {
    dispatch(timelineActions.clearEventsLoading({ id }));
    dispatch(timelineActions.clearEventsDeleted({ id }));
  }
}
```